src/en/help-oracle.md

```diff
@@ -0,0 +1,180 @@
+Title: Using the Oracle Compute Provider
+TODO: Test Oracle
+      Rewrite so its not so list heavy (especially Networks and spaces section)
+
+# Using the Oracle Compute Provider
+
+!!! Note:
+	Oracle Compute support is currently in [Juju 2.2 beta3][jujubeta] and
+        is experimental. There may be bugs. Work is continuing on improving the user
+        experience.
+
+To start using Juju with the [Oracle Compute Cloud][compute], some initial set up steps
+are required:
+
+1. Associate Ubuntu or Windows images from the marketplace with your Oracle account.
+   (this is an Oracle legal requirement).
```

Comment:
This parenthesis phrase should be part of the previous one or made into its own somehow.